src/Core/Components/Avatar/FluentAvatar.razor.cs

```diff
@@ -0,0 +1,124 @@
+
+/// <summary>
+/// The FluentButton component allows users to commit a change or trigger an action via a single click or tap and
+/// is often found inside forms, dialogs, drawers (panels) and/or pages.
+/// </summary>
+public partial class FluentAvatar : FluentComponentBase
+{
+    /// <summary />
+    protected string? ClassValue => DefaultClassBuilder.Build();
+
+    /// <summary />
+    protected string? StyleValue => DefaultStyleBuilder
+        .AddStyle("cursor", "pointer", OnClick.HasDelegate)
+        .Build();
+
+    /// <summary>
+    /// Optional activity indicator
+    /// </summary>
+    [Parameter]
+    public bool? Active { get; set; }
+
+    /// <summary>
+    /// Indicates the styled appearance of the avatar when the avatar is "active".
+    /// </summary>
+    [Parameter]
+    public AvatarActiveAppearance? ActiveAppearance { get; set; }
+
+    /// <summary>
+    /// Indicates the color of the avatar.
+    /// </summary>
+    [Parameter]
+    public AvatarColor? Color { get; set; }
+
+    /// <summary>
+    /// An avatar can display an icon. The icon will only be shown when there is no image or initials available.
+    /// </summary> 
+    [Parameter]
+    public Icon? Icon { get; set; }
+
+    /// <summary>
+    /// An avatar can display an image.
+    /// It is recommended to also include a name in addition to the image: the initials from the name are displayed while the image is loading, and the name makes the Avatar accessible to screen readers.
+    /// </summary>
+    [Parameter]
+    public string? Image { get; set; }
+
+    /// <summary>
+    /// Provide custom initials rather than one generated via the name
+    /// <para>It is usually not necessary to specify custom initials; by default they will be derived from the name prop, using the getInitials function.</para>
+    /// </summary>
+    [Parameter]
+    public string? Initials { get; set; }
+
+    /// <summary>
+    /// The name of the person or entity represented by this Avatar. This should always be provided if it is available.
+    /// The name is used to determine the initials displayed when there is no image.It is also provided to accessibility tools.
+    /// </summary>
+    [Parameter]
+    public string? Name { get; set; }
+
+    /// <summary>
+    /// Indicates the shape of the avatar.
+    /// </summary>
+    [Parameter]
+    public AvatarShape? Shape { get; set; }
+
+    /// <summary>
+    /// Size of the avatar in pixels.
+    /// <para>Size is restricted to a limited set of supported values recommended for most uses (see AvatarSizeValue) and based on design guidelines for the Avatar control.
+    /// Note: At size 16, if initials are displayed, only the first initial will be rendered.
+    /// If a non-supported size is needed, set size to the next-smaller supported size, and set width and height to override the rendered size.
+    /// For example, to set the avatar to 45px in size: style={{ width: '45px', height: '45px' }}</para>
```

Comment:
` style={{ width: '45px', height: '45px' }}` ??? What is it ? we are not in React